app/models/milestone.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Milestone < ApplicationRecord
```

Comment:
Do we want to start namespacing things? `Payment::Milestone` for example?

Reply:
Hmm, maybe. Let's go with `Finance` namespace, I think it is used in a few places now.